pkg/wallet/wallet_list_outputs_test.go

```diff
@@ -0,0 +1,253 @@
+package wallet_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/4chain-ag/go-wallet-toolbox/pkg/internal/fixtures"
+	"github.com/4chain-ag/go-wallet-toolbox/pkg/wallet/internal/testabilities"
+	sdk "github.com/bsv-blockchain/go-sdk/wallet"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWalletListOutputsArgsValidation(t *testing.T) {
+	errorTestCases := map[string]struct {
+		originator string
+		args       func() sdk.ListOutputsArgs
+	}{
+		"too long originator": {
+			originator: strings.Repeat("a", 251),
+			args: func() sdk.ListOutputsArgs {
```

Comment:
For defaults (without any modification), you could return the orifinal function itself. 

```suggestion
			args: fixtures.DefaultWalletListOutputsArgs,
```